<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1873</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1881</width>
     <height>911</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolButton" name="toolIngenieria">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Ingenieria De Producto</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/Ingenieria.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Comprar</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/autometrics.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>400</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="toolButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Rendimineto De Mercado Y Ventas</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/rendimiento de mercado y ventas.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="toolButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Gestión De La Cadena De Valor</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/gestion de la cadena de valores.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Innovación Y Tecnologia</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/innovacion y tecnologia.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="toolButton_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Riesgo Operacional</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/riesgo operacional.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>400</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Gestión De Flota</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/flota.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Capítal Humano Y Servicio</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/capital humano y servicio.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Inteligencia Financiera</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/Icon/inteligencia financiera.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>250</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Iconos/Iconos.qrc"/>
 </resources>
 <connections/>
</ui>
